Big-Spender/readme.md

```diff
@@ -112,7 +112,7 @@ INSERT YOUR QUERY HERE
 **You:** Then you need an extra clause. Here's the query:
 
 ```sql
-CREATE YOUR QUERY HERE
+SELECT to_char(date, 'YYYY-MM-DD') AS day, sum(amount) AS total FROM spends WHERE date IN ('2021-03-01', '2021-04-01') GROUP BY day;
```

Comment:
nice use of IN

why use to_char for the date here? it's fine, just curious.

Reply:
agree to_char is not necessary :)